packages/remix-react/server.tsx

```diff
@@ -31,6 +31,7 @@ export function RemixServer({
   }
 
   let { manifest, routeModules, criticalCss, serverHandoffString } = context;
+  debugger;
```

Comment:
@brophdawg11 was this left in by accident?

Reply:
Yes it was!  Thanks!